### PR TITLE
Use sinopia_api for export instead of trellis

### LINKED ISSUE #146: Update Exporter to use Sinopia API/MongoDB
The current Trellis export does not use the new Sinopia API with MongoDB. We need to update the Exporter to use the Sinopia API to extract the resources and resource templates, now in RDF, and generate a zip file of the contents per group. 

Requires a new Resources list endpoint  https://github.com/LD4P/sinopia_api/pull/22.

### PR DESCRIPTION
**Pull request by aaron-collier**
fixes #146 

Replaces #151

### CODE REVIEW

**Review by justinlittman — CHANGES_REQUESTED** [review]
Doesn't paging need to be handled?

**Comment by aaron-collier** [comment]
@justinlittman ah shoot, yeah, paging.... on it.

File: src/exporter.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  const savePathString = initAndGetSavePath(groupName, containingDir)
+  let done = false
+  let uri = [config.get('sinopia_api.basePath'), `/resource/?limit=${config.get('sinopia_api.query_limit')}&start=0&group=`, groupName].join('')
+
+  while (!done) {
+    let result_count = 0
+    await query(uri, { Accept: 'application/json' }).then((groupResources) => {
+      result_count = groupResources.data.length
+
+      groupResources.data.map((resource) => {
```
Comment (justinlittman) [inline src/exporter.js:55]: I think this is going to parallelize here.

File: src/exporter.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+}
+
+export const exportGroup = async (groupName, containingDir = '') => {
+  console.info(`beginning export of RDF from group: ${groupName}`)
+
+  const savePathString = initAndGetSavePath(groupName, containingDir)
+  let done = false
+  let uri = [config.get('sinopia_api.basePath'), `/resource/?limit=${config.get('sinopia_api.query_limit')}&start=0&group=`, groupName].join('')
+
+  while (!done) {
```
Comment (justinlittman) [inline src/exporter.js:50]: How do you exit the loop (or than L71)?

File: src/exporter.js (hunk 1 above)
Comment (aaron-collier) [reply src/exporter.js:55]: Can you explain why that would be an issue? This already has the resources returned as is writing to disk.

File: src/exporter.js (hunk 2 above)
Comment (aaron-collier) [reply src/exporter.js:50]: Yes, the line on 71. I could set done in the catch statements if that is a concern, but we can't break inside of an arrow function (results in `unsyntactic break` error.

Comment (justinlittman) [reply src/exporter.js:50]: Shouldn't it stop when it runs out of results?

Comment (aaron-collier) [reply src/exporter.js]: It did, if there is no next link or the results are less then expected. I updated this with a do/while for some clarity, hope that helps @justinlittman

**Comment by jermnelson** [comment]
@aaron-collier and @justinlittman how close are we to getting this PR merged? Thanks!

**Comment by justinlittman** [comment]
To the best I can tell all it needs is a little testing work, right @aaron-collier ?

**Comment by justinlittman** [comment]
Also, I'm unconcerned about test coverage and will merge when otherwise complete.

**Pull request merged**